src/services/ContentService.js

```diff
@@ -1108,7 +1108,10 @@ define(["structures/ContentCreateStruct", "structures/ContentUpdateStruct", "str
                     callback(error, views);
                     return;
                 }
-
+                if ( viewCreateStruct.getCriteria() && Object.keys(viewCreateStruct.getCriteria()).length !== 0 ) {
```

Comment:
it's just a matter of consistency with the others properties. This also ease the tests in PlatformUI where we have a clear API we can mock.